### ISSUE TITLE
New Repository 

### ISSUE THREAD

**Issue by Codex98** [Codex98/TheCommander_Game-#1]
alright. new repository. this is the old version of a old python input/output code which focuses on simple decision making operations in the context of military tactic. also added a new version; changes should be made to _this_ new code (theCommanderTwo), not the old one (theCommander_game)

**Comment by Codex98** [comment]
nevermind. found the new version and added it to the repository.

**Comment by Codex98** [comment]
objectives for this code: 
#object oriented; making profiles for users, and storing those profiles using pickling (?) 
#

**Comment by austBrink** [comment]
@Codex98 
Are we marking TheCommander_Game.py obsolete?

**Comment by Codex98** [comment]
@austBrink yes. its there for reference/practice (a guinea pig of sorts) if you need it. but again, all your operational changes are to be made to the thecommander2 file.

**Comment by austBrink** [comment]
Noted. I'll pass it on. thx
